sdk/storage/azblob/internal/shared/shared.go

```diff
@@ -133,6 +133,9 @@ func SerializeBlobTags(tagsMap map[string]string) *generated.BlobTags {
 	if tagsMap == nil {
 		return nil
 	}
+	if len(tagsMap) == 0 {
```

Comment:
Nit: with this change you can remove the above `nil` check as a `nil` map has a length of zero.

Reply:
fixed